### DISCUSSION TITLE
Doesn't module_server has setup-tables

### DISCUSSION THREAD

**Discussion by yahu1031** [serverpod/serverpod#177] [Q&A] [answered]
I tried creating a module, but I see the `setup-tables` file has not found an exception. And When I view the templates files, I see no `setup-tables` file. I think it is mandatory for modules too.

**Comment by anandsubbu007** [comment]
when change in module setup-tables is required to recreate table.

Create setup-table for both windows and mac. for now i think i supports to run in mac

**Comment by yahu1031** [comment]
Modules are not supposed to have tables or DB. So no need for setup-tables script. (Noob me).

**Comment by vlidholt** [comment, marked as answer]
Modules can have database tables. We are much improving the workflow for this in version 1.2.
